cmake/modules/Findthrift.cmake

```diff
@@ -0,0 +1,93 @@
+# https://github.com/snikulov/cmake-modules/blob/master/FindThrift.cmake
```

Comment:
i cannot find the license of this file in https://github.com/snikulov/cmake-modules. and its coding style is quite different from ours. i'd suggest create our own.